lec16-data-viz.Rmd

```diff
@@ -1,4 +1,97 @@
 ---
-title: "More on data visualizations (plus project work)"
+title: "More on data visualizations and the git workflow (plus project work)"
+author: Joel Ostblom
 ---
 
+First, let's create a chat room for your repository. We will use gitter, since
+it has good integrations with GitHub. Got to http://gitter.im/, login with your
+GitHub account and click the plus sign to create a new chat room. Browse to
+your repository, and add the members of your team. Since your repository is
+public, this chatroom will also be public (so you might not want to share
+personal information, and keep your discussions on topic, just as if this was
+a real-world work project).
+
+Discussions in the chat room are good to crystallize what tasks need to be
+tackled next. Once you have identified a specific task, open an issue about it
+on GitHub to track the progress of that particular tasks. You should create
+issues for your planned analyses, writing, literature review, etc. Then
+schedule those into your day and close them when you are done with that task.
+It is similar to a powerful ToDo manager for your project. In general, you
+should create issues in the main repository, not in your own fork of the
+repository. Let's create a new issue based on our discussion, assign the person
+we think should work on it and then review the workflow for addressing the issue.
+
+*Create new issue about adding a file to the sandbox repo*
```

Comment:
Why not a heading style? Is this an instruction for you?

Reply:
Yes, that is an instruction to me. But it is unnecessary, I changed.